static/src/js/components/AccessMethod/AccessMethod.js

```diff
@@ -530,6 +720,26 @@ export class AccessMethod extends Component {
                     </ProjectPanelSection>
                   )
                 }
+                {
+                  // Show Harmony method description
+                  harmonyTypeSelected && (
+                    <>
+                      <p />
```

Comment:
Lets try to simplify the number of HTML elements we're using here I think some of them are unnecessary. I think this would be wrapped by a `div` instead of a fragment